plugins/modules/system/selogin.py

```diff
@@ -207,7 +211,7 @@ def main():
         argument_spec=dict(
             ignore_selinux_state=dict(type='bool', default=False),
             login=dict(type='str', required=True),
-            seuser=dict(type='str'),
+            seuser=dict(type='str', required=True),
```

Comment:
This is wrong. `seuser` is only required if `state=present`, which is correctly mapped to a `required_if` below. The documentation must be fixed accordingly.